workspace.go

```diff
@@ -1062,7 +1062,8 @@ func (o WorkspaceCreateOptions) valid() error {
 	if o.AgentPoolID == nil && (o.ExecutionMode != nil && *o.ExecutionMode == "agent") {
 		return ErrRequiredAgentPoolID
 	}
-	if o.TriggerPrefixes != nil && o.TriggerPatterns != nil {
+	if o.TriggerPrefixes != nil && len(o.TriggerPrefixes) > 0 &&
```

Comment:
Curious! Why do we validate the length of these attributes for the WorkspaceCreateOptions but not the WorkspaceUpdateOptions?

Reply:
The reason being - I was hasty and forgot to add changes to the update part 🤦🏻‍♂️ 
Good catch!

Reply:
a290810956ef053e4c08c864bcf0b9c65293346a